doc/quick-start.rst

```diff
@@ -6,10 +6,148 @@ This document gives simple usage examples of Dune. You can also look at
 `examples <https://github.com/ocaml/dune/tree/master/example>`__ for complete
 examples of projects using Dune.
 
+Initializing projects
+=====================
+
+The following subsections illustrate basic usage of the ``dune init proj``
```

Comment:
The following subsections illustrate basic usage of the ``dune init proj``
subcommand. For more documentation, see :ref:`initializing_components` and the
inline help available from ``dune init --help``.

Reply:
(just a comma after "Fore more documentation")